packages/slice-machine/src/legacy/lib/models/common/widgets/Group/ListItem/index.jsx

```diff
@@ -171,15 +170,35 @@ export const CustomListItem = ({
         {...rest}
         HintElement={HintElement}
         CustomEditElements={[
-          <Button
-            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-member-access
-            key={`custom-edit-element-${groupItem.key}`}
-            mr={2}
-            variant="buttons.darkSmall"
-            onClick={() => setSelectMode(true)}
-          >
-            Add Field
-          </Button>,
+          <AddFieldDropdown
+            key="add-field-dropdown"
+            disabled={newFieldData !== null}
+            onSelectField={onSelectFieldType}
+            fields={
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+              widgetsArray
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+                .filter((e) => e)
```

Comment:
Tiny improvement suggestion 🧼

```suggestion
                .filter(Boolean)
```

Reply:
Yeah better good catch, copy-paste this old code, when working on JS file, not trying to really improve things 😅 
I'm mostly focusing on extracting a little by little code from legacy to new folders.

Reply:
Done 🙂